docs/2.8.0/docs/canton/usermanual/manage_domain_entities.rst

```diff
@@ -178,3 +178,11 @@ dynamically onboarding new sequencers (supported by Fabric and Ethereum sequence
    :start-after: user-manual-entry-begin: DynamicallyOnboardSequencerWithSeparateConsoles
    :end-before: user-manual-entry-end: DynamicallyOnboardSequencerWithSeparateConsoles
    :dedent:
+
+A newly onboarded sequencer only serves events more recent than the "onboarding snapshot". In addition some events may belong to transactions initiated before
+the sequencer was onboarded, but the sequencer is not in a position to sign such events and replaces them with "tombstones". If a participant (or mediator)
+connects to a newly onboarded sequencer too soon and the subscription encounters a tombstone, the sequencer subscription aborts with a "FAILED_PRECONDITION" error
+specifying "InvalidCounter" or "SEQUENCER_TOMBSTONE_ENCOUNTERED". If this occurs, the participant or mediator should connect to another sequencer with a longer
+history of sequenced events before switching to the newly onboarded sequencer. To avoid such errors the best practice is to wait at least for the "maximum decision time"
```

Comment:
Nit: we should keep _time_ to denote points in time. Switch to "maximum decision duration" or similar ?

Reply:
Follow-up PR at: https://github.com/digital-asset/docs.daml.com/pull/514